CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+[npm history][1]
+
+[1]: https://www.npmjs.com/package/@google-cloud/talent?activeTab=versions
+
+## v0.1.0
+
+03-12-2019 21:57 PDT
+
+First release!
```

Comment:
my colleague CJ used to like putting `reverts prior commit` for the first commit, just to troll people.

Reply:
I might steal that